src/components/content/spoiler.tsx

```diff
@@ -31,3 +31,63 @@ export function Spoiler({ body, title, path }: SpoilerProps) {
     </div>
   )
 }
+
+function SpoilerTitle({
```

Comment:
Sorry, same here. 
If you want to introduce exporting multiple components from one file we would have to discuss this first.

Reply:
It's not exported though...